### DISCUSSION TITLE
Suggestion: More realistic demand

### DISCUSSION THREAD

**Discussion by lordjoda** [voxeltycoon/issues#377] [General]
The current system forces you to choose to either keep supplying a buisness with a "base" product, or letting it go bancrupt to chance spawn a new  buisness that accepts a "manufactured" good, to increase the income, but otherwise afaik, the city does not care what kind of goods are delivered. This in my opinion feels bad, since I don't want to let a buisness go bancrupt, but want to have to deliver more and different goods to the cities to make them grow.
I therefore would suggest a (probably big) overhaul of the curent system.

1.  Categorization of Goods: 
There should be different kind of goods: Goods that are just for creating new goods (like iron ore), that can not be sold in towns, goods that can be sold in towns and maybe a combination of both.
2.  Grouping of wares in buisnesses:
Buisnesses should not only accept one type of good, but multiple in the same category (like household, electronics etc). 
3.  Tying demand to town size:
If a town has reached a spezific size, it will start requiring additional goods for it to grow further. If a good is no longer supplied, the town will start shrinking, or not continue to grow.
4.  Tying buisness creation to town size:
Together with the demand of the town, new buisnesses should only be created, if the town demands the goods the buiness would accept (e.g. no electronics store for small towns etc.). If a town should have multiple buisnesses of the same type is of course something that can be considered, but I have no preference either way.

These changes of course require the player to create complex production chains, but also require him to keep producing "early" wares, to keep the satisfy the demand of the town.
All of this of course also influences the demand of resources and might require a rebalance about how many ores etc are in a resource node, but in my opinion it can lead to a more challenging gameplay. 
I understand that this touches a very central part of the game, so I can understand if this is not the direction you want to go with the game.

**Comment by matthewfarmery** [comment]
A topic like this is on the steam forum. 

https://steamcommunity.com/app/732050/discussions/0/4679778856509241010/

This is what I wrote

.........................

But the problem is, in this game, the demands can get a bit much, sometimes demanding stuff your not ready for. Which while might be a challenge. You might not have either the resources or the money to research the tech. Small villages could be speciallised in doing just wood or just stone. If you either have them in the starting zone or you unlock in another region could be random. Not all villages should grow. Or it might be a way to control the growth of a village.

Also if the village doesn't want any of your products, but products from another city, you could fulfil that and encourage players to to connect towns and cities together.

So villages start of either been fixed, or mixed. so also maybe fixed or mixed depending on the location, if neat a forest for example, they might only want to have wood stuff, cabins, wooden buildings etc. but they lack the resources to do that. But in turn they could maybe mine for gold, or some other resource that you can't do. So you help them, they help you. There are a few ways this could work, and why a village might want to be locked to a certain resource or resources. (especailly if they are so far away from anything that could generate power, so stuff like radios won't be of any use to them. (or TVs)

It would make the game more interesting, and maybe add different challenges to the game. I do hope we see something like this at some point.
